src/main/python/org/broadinstitute/hellbender/gcnvkernel/models/model_denoising_calling.py

```diff
@@ -25,6 +25,8 @@
 
 _logger = logging.getLogger(__name__)
 
+_eps = 1E-10
```

Comment:
Could you extract this constant here and in `model_ploidy.py` to a separate file? Maybe it's also would be worth promoting this to a config parameter so we can experiment with it in the future

Reply:
Just put it in models.commons.  Maybe we can think about exposing it if more NaNs crop up.